azero/contracts/most/lib.rs

```diff
@@ -464,22 +477,24 @@ pub mod most {
 
         /// Request payout of rewards for signing & relaying cross-chain transfers.
         ///
-        /// Can be called by anyone on behalf of the committee member.
+        /// Reverts if `member_id` account is not in the committee with `committee_id`
         #[ink(message)]
         pub fn payout_rewards(
             &mut self,
             committee_id: CommitteeId,
             member_id: AccountId,
         ) -> Result<(), MostError> {
             self.ensure_not_halted()?;
+            self.only_committee_member(committee_id, member_id)?;
```

Comment:
Now this check seems unnecessary

Reply:
an early return, saves gas - but yeah we can remove. Let me know with :+1: / :-1:

Reply:
Ah, ok, makes sense, we ca leave it as is